package.json

```diff
@@ -10,33 +10,26 @@
   ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/Joystream/joystream.git"
+    "url": "https://github.com/Joystream/atlas.git"
   },
   "bugs": {
-    "url": "https://github.com/Joystream/joystream/issues"
+    "url": "https://github.com/Joystream/atlas/issues"
   },
   "scripts": {
-    "bundle-stats": "webpack-bundle-analyzer dist/bundle-stats.json -m static -r dist/stats.html",
-    "start": "react-app-rewired start",
-    "dev": "yarn start",
-    "build": "react-app-rewired build --stats && yarn bundle-stats",
-    "eject": "react-app-rewired eject",
+    "start": "vite",
+    "dev": "vite",
+    "build": "vite build",
+    "storybook": "start-storybook -p 6006",
```

Comment:
storybook not working for me. I'm getting: 
![image](https://user-images.githubusercontent.com/51168865/131965086-8e50913d-4c32-483d-90e7-df7527e8ca08.png)

Reply:
Works great now, it's super fast 🚀